packages/terra-list/src/ListSubsection.jsx

```diff
@@ -59,7 +66,7 @@ const ListSubsection = ({
 
   return (
     <React.Fragment>
-      <SubsectionHeader {...customProps} isCollapsible={isCollapsible} isCollapsed={isCollapsed} />
+      <SubsectionHeader {...customProps} role={role} isCollapsible={isCollapsible} isCollapsed={isCollapsed} />
```

Comment:
This isn't needed, as customProps will already spread the value.

Reply:
Changed 15166a3